test/val/val_decoration_test.cpp

```diff
@@ -8003,6 +8003,7 @@ TEST_F(ValidateDecorations, WorkgroupBlockVariableWith16BitType) {
                OpCapability Shader
                OpCapability Float16
                OpCapability Int16
+               OpCapability WorkgroupMemoryExplicitLayoutKHR
```

Comment:
this test was failing because of grammar bug not passing implicit declare

https://github.com/KhronosGroup/SPIRV-Headers/pull/388